cypress/e2e/title-level-request/check-user-can-see-open-not-yet-filled-accordeon.cy.js

```diff
@@ -0,0 +1,205 @@
+      Requests.createNewRequestViaApi({
+        fulfillmentPreference: FULFILMENT_PREFERENCES.HOLD_SHELF,
+        holdingsRecordId: testData.holdingTypeId,
+        instanceId: instanceData.instanceId,
+        item: { barcode: testData.itemBarcode },
+        itemId: instanceData.itemId[0],
+        pickupServicePointId: testData.userServicePoint.id,
+        requestDate: new Date(),
+        requestLevel: REQUEST_LEVELS.ITEM,
+        requestType: REQUEST_TYPES.PAGE,
+        requesterId: userData.userId,
+      }).then((request) => {
+        requestIds.push(request.body.id);
+      });
+      Requests.createNewRequestViaApi({
+        fulfillmentPreference: FULFILMENT_PREFERENCES.HOLD_SHELF,
+        instanceId: instanceData.instanceId,
+        pickupServicePointId: testData.userServicePoint.id,
+        requestDate: new Date(),
+        requestLevel: REQUEST_LEVELS.TITLE,
+        requestType: REQUEST_TYPES.HOLD,
+        requesterId: userForTLR.userId,
+      }).then((request) => {
+        requestIds.push(request.body.id);
+      });
+      cy.login(userData.username, userData.password, {
+        path: TopMenu.requestsPath,
+        waiter: Requests.waitLoading,
+      });
+    });
+  });
+
+  after('Delete test data', () => {
+    cy.loginAsAdmin({
+      path: SettingsMenu.circulationTitleLevelRequestsPath,
+      waiter: TitleLevelRequests.waitLoading,
+    });
+    cy.wrap(requestIds).each((id) => {
+      Requests.deleteRequestViaApi(id);
+    });
+    InventoryInstances.deleteInstanceAndHoldingRecordAndAllItemsViaApi(testData.itemBarcode);
+    RequestPolicy.deleteViaApi(requestPolicyBody.id);
+    CirculationRules.deleteRuleViaApi(testData.addedRule);
+    cy.deleteLoanType(testData.loanTypeId);
+    UserEdit.changeServicePointPreferenceViaApi(userForTLR.userId, [testData.userServicePoint.id]);
+    UserEdit.changeServicePointPreferenceViaApi(userData.userId, [testData.userServicePoint.id]);
+    ServicePoints.deleteViaApi(testData.userServicePoint.id);
+    Users.deleteViaApi(userForTLR.userId);
+    Users.deleteViaApi(userData.userId);
+    PatronGroups.deleteViaApi(patronGroup.id);
+    Location.deleteViaApiIncludingInstitutionCampusLibrary(
+      testData.defaultLocation.institutionId,
+      testData.defaultLocation.campusId,
+      testData.defaultLocation.libraryId,
+      testData.defaultLocation.id,
+    );
+    TitleLevelRequests.changeTitleLevelRequestsStatus('forbid');
+  });
+
+  it(
+    'C347891 Check that user can see Open - Not yet filled accordion (vega) (Taas)',
+    { tags: ['extendedPath', 'vega'] },
+    () => {
+      Requests.findCreatedRequest(instanceData.title);
+      Requests.selectFirstRequest(instanceData.title);
+      RequestDetail.requestQueueOnInstance(instanceData.title);
+      RequestDetail.verifyHeaders(instanceData.title);
+      RequestDetail.verifyAllQueueAccordeonsDisplayed();
+      RequestDetail.verifyNotFilledYetAccordionHeaders();
```

Comment:
Please try to use RequestDetail.verifyRequestQueueColumnsPresence();
this could prevent unnecessary methods from being created.